health/health.go

```diff
@@ -161,16 +177,31 @@ type warnOptFunc func(*Warnable)
 
 func (f warnOptFunc) mod(w *Warnable) { f(w) }
 
-// Warnable is a health check item that may or may not be in a bad warning state.
-// The caller of NewWarnable is responsible for calling Tracker.SetWarnable to update the state.
+// WarnableCode is a string that is used to distinguish each Warnable from others.
+type WarnableCode string
+
+// A Warnable is an entity we warn the user about by means of a Warning.
+// In most cases, Warnables are components of the backend (for instance, "DNS" or "Magicsock").
+// Warnables are similar to the Subsystem type previously used in this package, but they provide
+// a unique identifying code for each Warnable, along with more metadata.
+// A Warnable may or may not be in an error state, depending on whether the Tracker knows about
+// one or more Warnings affecting the Warnable.
+// The caller of NewWarnable is responsible for calling Tracker.AddWarning() or
+// Tracker.RemoveAllWarnings() to update the warnings known for each Warnable.
 type Warnable struct {
+	Code WarnableCode // identifies this Warnable, and can be mapped to a user-displayable localized string
+
 	debugFlag string // optional MapRequest.DebugFlag to send when unhealthy
```

Comment:
get rid of this entirely? report everything eventually?

Reply:
meanwhile this should only be used for check-ip-forwarding or whatever it's called.

Reply:
TODO in next PR, seems like it's a massive delete